nautobot/core/templates/generic/object_list.html

```diff
@@ -104,17 +104,39 @@
             </ul>
         </div>
     {% endif %}
-    {% if permissions.add and 'add' in action_buttons %}
-        {% add_button content_type.model_class|validated_viewname:"add" %}
-    {% endif %}
+    <div class="btn-group">
+        {% if permissions.add and 'add' in action_buttons %}
+            {% add_button content_type.model_class|validated_viewname:"add" %}
+            <button type="button" id="actions-dropdown" class="btn btn-primary dropdown-toggle" data-toggle="dropdown">
+                <span class="caret"></span>
+                <span class="sr-only">Toggle Dropdown</span>
+            </button>
+        {% else %}
+            <button type="button" id="actions-dropdown" class="btn btn-primary dropdown-toggle" data-toggle="dropdown">
+                Actions <span class="caret"></span>
+            </button>
+        {% endif %}
+        <ul class="dropdown-menu pull-right" role="menu">
+            {% if permissions.add and 'import' in action_buttons %}
+                {% block import_list_element %}
+                    {% job_import_button content_type list_element=True%}
```

Comment:
Yes, we do use it in core for the Device-Type and Module-Type List Views. We also put it in our Apps doc, so there could be others that use it. I left the import_button and export_button blocks in place in case Apps do use this feature.